src/http_rest_api_handler.cpp

```diff
@@ -499,7 +499,7 @@ Status HttpRestApiHandler::processV3(const std::string_view uri, const HttpReque
         serverReaderWriter->OverwriteResponseHeader("Connection", "keep-alive");
         auto status = executor->inferStream(request, *serverReaderWriter);
         if (!status.ok()) {
-            sendErrorImpl(status.string(), *serverReaderWriter);
+            sendErrorImplWithCode(status.string(), 400, *serverReaderWriter);
```

Comment:
There are multiple places where we send an error using `sendErrorImpl` - why do we only return with status here? Please check if we need to also return with status in all other cases (check all usages of sendErrorImpl)